src/expression/parse.js

```diff
@@ -664,18 +665,24 @@ export const createParse = /* #__PURE__ */ factory(name, dependencies, ({
 
     const node = parseConditional(state)
 
-    if (state.token === '=') {
+    if (state.token === '=' || state.token === ':=') {
+      const assignString = state.token
+
       if (isSymbolNode(node)) {
         // parse a variable assignment like 'a = 2/3'
         name = node.name
         getTokenSkipNewline(state)
         value = parseAssignment(state)
-        return new AssignmentNode(new SymbolNode(name), value)
+        const assignmentNode = new AssignmentNode(new SymbolNode(name), value)
+        assignmentNode.assignString = assignString
+        return assignmentNode
```

Comment:
@josdejong please let me know what you think about this, because it does not feel totally right to me as well doing it by assigning the assignString directly to the object. Maybe its better to provide it as param? but then some more checks in the constructor would be needed, because of `node.index` which is optional. Or as a separate chainable method, like 

```
return new AssignmentNode(...)
  .setAssignString(assignString)
```

Again, all this is actually because of the `:=` assignString for the toTex representation.